<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>grade_calculator</class>
 <widget class="QMainWindow" name="grade_calculator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>grade_calculator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="titleLabel">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>20</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Grade Calculator</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_8">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>170</y>
      <width>341</width>
      <height>91</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Midterm 2</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_12">
       <item>
        <widget class="QSlider" name="horizontalSlider_11">
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="sb10">
         <property name="maximum">
          <number>100</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_9">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>300</y>
      <width>341</width>
      <height>91</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_5" stretch="0,0">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Final</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_13">
       <item>
        <widget class="QSlider" name="horizontalSlider_12">
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="sb11">
         <property name="maximum">
          <number>100</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>480</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QLabel" name="gradeLabel">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Overall Grade:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="overallGrade">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>371</width>
      <height>471</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="HW1">
         <property name="text">
          <string>HW1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="horizontalSlider">
         <property name="maximum">
          <number>20</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="sb1">
         <property name="maximum">
          <number>20</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="HW2">
         <property name="text">
          <string>HW2</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="horizontalSlider_3">
         <property name="maximum">
          <number>20</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="sb2">
         <property name="maximum">
          <number>20</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="HW3">
         <property name="text">
          <string>HW3</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="horizontalSlider_4">
         <property name="maximum">
          <number>20</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="sb3">
         <property name="maximum">
          <number>20</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="HW4">
         <property name="text">
          <string>HW4</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="horizontalSlider_5">
         <property name="maximum">
          <number>20</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="sb4">
         <property name="maximum">
          <number>20</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="HW5">
         <property name="text">
          <string>HW5</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="horizontalSlider_6">
         <property name="maximum">
          <number>20</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="sb5">
         <property name="maximum">
          <number>20</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QLabel" name="HW6">
         <property name="text">
          <string>HW6</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="horizontalSlider_7">
         <property name="maximum">
          <number>20</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="sb6">
         <property name="maximum">
          <number>20</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QLabel" name="HW7">
         <property name="text">
          <string>HW7</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="horizontalSlider_8">
         <property name="maximum">
          <number>20</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="sb7">
         <property name="maximum">
          <number>20</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_10">
       <item>
        <widget class="QLabel" name="HW8">
         <property name="text">
          <string>HW8</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="horizontalSlider_9">
         <property name="maximum">
          <number>20</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="sb8">
         <property name="maximum">
          <number>20</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>80</y>
      <width>341</width>
      <height>91</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Midterm 1</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_11">
       <item>
        <widget class="QSlider" name="horizontalSlider_10">
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="sb9">
         <property name="maximum">
          <number>100</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>480</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item>
      <widget class="QRadioButton" name="radioButton">
       <property name="text">
        <string>Shema A</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_2">
       <property name="text">
        <string>Schema B</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>horizontalSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>sb1</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>291</x>
     <y>127</y>
    </hint>
    <hint type="destinationlabel">
     <x>373</x>
     <y>130</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sb1</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>367</x>
     <y>123</y>
    </hint>
    <hint type="destinationlabel">
     <x>252</x>
     <y>135</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_3</sender>
   <signal>valueChanged(int)</signal>
   <receiver>sb2</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>293</x>
     <y>187</y>
    </hint>
    <hint type="destinationlabel">
     <x>362</x>
     <y>191</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sb2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_3</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>364</x>
     <y>187</y>
    </hint>
    <hint type="destinationlabel">
     <x>293</x>
     <y>188</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_4</sender>
   <signal>valueChanged(int)</signal>
   <receiver>sb3</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>309</x>
     <y>244</y>
    </hint>
    <hint type="destinationlabel">
     <x>369</x>
     <y>253</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sb3</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_4</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>360</x>
     <y>244</y>
    </hint>
    <hint type="destinationlabel">
     <x>287</x>
     <y>244</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_5</sender>
   <signal>valueChanged(int)</signal>
   <receiver>sb4</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>306</x>
     <y>302</y>
    </hint>
    <hint type="destinationlabel">
     <x>373</x>
     <y>311</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sb4</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_5</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>370</x>
     <y>301</y>
    </hint>
    <hint type="destinationlabel">
     <x>281</x>
     <y>316</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_6</sender>
   <signal>valueChanged(int)</signal>
   <receiver>sb5</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>308</x>
     <y>367</y>
    </hint>
    <hint type="destinationlabel">
     <x>386</x>
     <y>370</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sb5</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_6</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>362</x>
     <y>362</y>
    </hint>
    <hint type="destinationlabel">
     <x>260</x>
     <y>364</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_7</sender>
   <signal>valueChanged(int)</signal>
   <receiver>sb6</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>288</x>
     <y>421</y>
    </hint>
    <hint type="destinationlabel">
     <x>377</x>
     <y>422</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sb6</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_7</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>373</x>
     <y>434</y>
    </hint>
    <hint type="destinationlabel">
     <x>242</x>
     <y>424</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_8</sender>
   <signal>valueChanged(int)</signal>
   <receiver>sb7</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>305</x>
     <y>479</y>
    </hint>
    <hint type="destinationlabel">
     <x>372</x>
     <y>484</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sb7</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_8</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>354</x>
     <y>493</y>
    </hint>
    <hint type="destinationlabel">
     <x>226</x>
     <y>487</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_9</sender>
   <signal>valueChanged(int)</signal>
   <receiver>sb8</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>306</x>
     <y>541</y>
    </hint>
    <hint type="destinationlabel">
     <x>372</x>
     <y>541</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sb8</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_9</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>360</x>
     <y>554</y>
    </hint>
    <hint type="destinationlabel">
     <x>276</x>
     <y>540</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_10</sender>
   <signal>valueChanged(int)</signal>
   <receiver>sb9</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>688</x>
     <y>172</y>
    </hint>
    <hint type="destinationlabel">
     <x>752</x>
     <y>172</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sb9</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_10</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>744</x>
     <y>163</y>
    </hint>
    <hint type="destinationlabel">
     <x>674</x>
     <y>169</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_11</sender>
   <signal>valueChanged(int)</signal>
   <receiver>sb10</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>682</x>
     <y>264</y>
    </hint>
    <hint type="destinationlabel">
     <x>746</x>
     <y>264</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sb10</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_11</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>733</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>672</x>
     <y>253</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_12</sender>
   <signal>valueChanged(int)</signal>
   <receiver>sb11</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>706</x>
     <y>388</y>
    </hint>
    <hint type="destinationlabel">
     <x>755</x>
     <y>389</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sb11</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_12</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>751</x>
     <y>394</y>
    </hint>
    <hint type="destinationlabel">
     <x>632</x>
     <y>392</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
